### ISSUE TITLE
PonsTranslator raises IndexError translating arabic to english

### ISSUE THREAD

**Issue by djhenderson** [nidhaloff/deep_translator#11]
* deep_translator version: 1.1.5
* Python version: Python 3.9.0b5
* Operating System: Win 10 Pro

### Description

I am trying to translate an arabic sentence to english.
I believe this was working, or at least not raising an exception with version 1.1.4.
The arabic sentence `آخُذ اَلْباص.` should translate to something like `I take the bus.`.

Note: this form is rendering the FULL_STOP at the right end of the string. Several other LTR renders will place it in the correct position at the left end of the string.

### What I Did

```
> py -3.9
Python 3.9.0b5 (tags/v3.9.0b5:8ad7d50, Jul 20 2020, 18:35:09) [MSC v.1924 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import deep_translator
>>> deep_translator.__version__
'1.1.5'
>>> from deep_translator import PonsTranslator
>>> t = PonsTranslator(source='arabic', target='english')
>>> t.translate('آخُذ اَلْباص.')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\Doug\AppData\Local\Programs\Python\Python39\lib\site-packages\deep_translator\pons.py", line 81, in translate
    return word_list if return_all else word_list[0]
IndexError: list index out of range
>>>
```

**Comment by djhenderson** [comment]
Here are the details of the string I am trying to translate:
```
ord     cat     bid     co      mi      decomp          ch      name
0622    Lo      AL        0      0      0627 0653       آ      ARABIC LETTER ALEF WITH MADDA ABOVE
062e    Lo      AL        0      0                      خ      ARABIC LETTER KHAH
064f    Mn      NSM      31      0                      ُ      ARABIC DAMMA
0630    Lo      AL        0      0                      ذ      ARABIC LETTER THAL
0020    Zs      WS        0      0                              SPACE
0627    Lo      AL        0      0                      ا      ARABIC LETTER ALEF
064e    Mn      NSM      30      0                      َ      ARABIC FATHA
0644    Lo      AL        0      0                      ل      ARABIC LETTER LAM
0652    Mn      NSM      34      0                      ْ      ARABIC SUKUN
0628    Lo      AL        0      0                      ب      ARABIC LETTER BEH
0627    Lo      AL        0      0                      ا      ARABIC LETTER ALEF
0635    Lo      AL        0      0                      ص      ARABIC LETTER SAD
002e    Po      CS        0      0                      .       FULL STOP
```

**Comment by nidhaloff** [comment]
Thanks for opening this issue and the detailed description. I don't think this issue is caused by the package per se. I guess the PONS translator https://en.pons.com/translate can't translate that sentence from arabic to english too. However, it's very interesting

The deep-translator package is using the translators under the hood by sending API requests. So if something can't be done with the PONS translator, then it can't be done using the deep-translator package too (I'm just explaining however, I'm sure you know that).

I should at least catch the exception and provide a good message for the user that no translation is provided. From my research and experience, I find PONS and Linguee translators better suitable for translating single words, whereas the google or mymemory translators perform better when translating complete sentences.

**Comment by djhenderson** [comment]
The error remains:
```
  File "C:\Users\Doug\AppData\Local\Programs\Python\Python39\lib\site-packages\deep_translator\pons.py", line 81, in translate
    return word_list if return_all else word_list[0]
```
This coded fails when `return_all` is False, and word_list is an empty list.

The more basic problem is that the PONS online translator does not translate 'arabic' to anything other than 'german'.

This makes me suspect that the dictionary API does not either. When I run it on each word in a sentence instead of the who;e sentence, as I do with the other translators, it fails to translates a lot of words, but for a few it returns some strange stuff, for example:

```
source_word='صَفّ'
target_word=['column ', 'antirrhinum ', 'Canadian ', 'aristocracy ', 'polytheistic ', 'sabbatic ', 'collors ', 'Jewish ', 'scurvy ', 'cordon ', 'tin ', 'canary ', 'simile ', 'quick-lunch ']
```
Google translates that word as 'class' as in a university class.

**Comment by nidhaloff** [comment]
@djhenderson Excatly, That error was fixed in release 1.1.6 thanks to you who noticed this issue. The actual release is 1.1.7 where I added the simple and batch language detection features ;)

Like you said, the pons translator have some limitations especially with the arabic language. Therefore, you may try to use google translate or mymemory. The example with the word **source_word='صَفّ'** is a little bit tricky because that word means column in the arabic language.
However, the other synonyms returned by pons looks weird but as I said, this have nothing to do with the package since it is a limitation of the pons translator itself.

**Comment by nidhaloff** [comment]
I will close the issue since it was resolved in the new release ;)
